ush/load_fv3gfs_modules.sh

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env bash
 
 ###############################################################
-if [[ "${DEBUG_WORKFLOW:-NO}" == "NO" ]]; then
```

Comment:
This is incorrect. Please undo this change.

Reply:
I undo this change.